src/internal/connector/sharepoint/collection.go

```diff
@@ -271,13 +271,23 @@ func (sc *Collection) retrievePages(
 	var (
 		errs    error
 		metrics numMetrics
+		root    string
 	)
 
 	betaService := sc.betaService
 	if betaService == nil {
 		return metrics, fmt.Errorf("beta service not found in collection")
 	}
 
+	parent, err := sapi.GetSiteLite(ctx, sc.service, sc.fullPath.ResourceOwner())
+	if err != nil {
+		return metrics, err
+	}
+
+	if parent.GetWebUrl() != nil {
```

Comment:
ptr.Val here?  If nil it'll hold an empty string either way.

Reply:
Resolved with [3fd5fa8](https://github.com/alcionai/corso/pull/2461/commits/3fd5fa8da842e2135c580f68ec21ba86b4d24879)